### PR TITLE
Routed class cards without any CSS styling

### PR DESCRIPTION
**Pull request by thaowii**
- Made class cards clickable
- Routed class cards to the respective course page
- URL for the respective class card is ".../courses/classnumber" (If you click on CSE 403 it will take you to ".../courses/403")
- Created javascript to make api call to fetch data for the class card that was clicked

Testing:
- Clicked on multiple class cards and saw that the correct data is consistently fetched and rendered
- Played with URLs to make sure that similar URLs do not load the data (If I typed in ".../403" instead of ".../courses/403" the site will load nothing)

### CODE REVIEW

File: app/src/App/App.tsx [hunk 1]
```diff
@@ -18,10 +19,12 @@ function App() {
             <Route
               path={"/" + classPrefix + classNumber}
               element={<ClassList classLevelNumber={classNumber[0]} />}
-            />
+            >
+            </Route>
```
Comment (celestinebuendia) [inline app/src/App/App.tsx:23]: Why change this route from self-closing to not, when the others aren't?

**Review by celestinebuendia — CHANGES_REQUESTED** [review]
Looks great! One question though! Sorry!

(But do want to merge this soon, since I want that API call for the review pages)

File: app/src/App/App.tsx (hunk 1 above)
Comment (thaowii) [reply app/src/App/App.tsx]: Oh woops I was playing around with stuff and forgot to change that back. Fixed!

**Review by celestinebuendia — APPROVED** [review]
Thanks!

**Pull request merged**